common/models/wordbreakers/build.sh

```diff
@@ -25,11 +26,18 @@ builder_describe_outputs \
 
 builder_parse "$@"
 
+function do_configure() {
+  verify_npm_setup
+
+  tsc -b src/data-compiler/tsconfig.json
+  node ./build/obj/data-compiler/index.js
```

Comment:
As the Unicode version we utilize is rarely ever going to change, I felt this makes sense as a `configure` step.

If we do remove the checked in `data.ts` (and swap to a `data.inc.ts`), I'll change the `builder_describe_outputs` to refer to `data.inc.ts`.  The `.inc.ts` version did not exist before, while the plain `.ts` version did.

Reply:
As a rule of thumb, `configure` and `build` actions should never make the repo dirty. So either this should be a separate action, or else we do not check in the data file (in which case under 'configure' is good).

My preference: don't check in files we can easily generate. And then, can you add a comment above the `tsc -b` line to describe what file is generated here and why it's here and not in `build`? (and add data.inc.ts to .gitignore ofc)

Reply:
> That said, the data hasn't been updated in a while, so keeping it here for now, at this stage of the reviewing process, highlights what updating the data will change.

I'm perfectly happy to drop the file in this PR; I just wanted to make sure that the fact we're doing an update here is quite visible and explicit.  We're now a bit further along in the "reviewing process".

> My preference: don't check in files we can easily generate.

That would be the long-term idea.